src/components/views/homepage/HomePurpleSection.tsx

```diff
@@ -187,6 +187,7 @@ const ArcSmall = styled(Arc)`
 const ForMakersGivers = styled(Row)`
 	margin-top: 107px;
 	margin-bottom: 50px;
+	justify-content: space-between;
```

Comment:
there is too much space between them. maybe it's a better idea too set a gap between the two sections instead of `space-between`

<img width="1242" alt="image" src="https://user-images.githubusercontent.com/73441857/161402181-509b5ac4-6a27-42c3-b5f4-368dbd3fd34b.png">